tests/ansible_collections/roles/get-test-vars/tasks/get-test-vars.yml

```diff
@@ -8,7 +8,7 @@
 
 - name: Download the Satellite file
   get_url:
-    url: https://gitlab.cee.redhat.com/oamg/convert2rhel/convert2rhel-secrets/-/raw/main/.env_sat_reg
+    url: https://gitlab.cee.redhat.com/oamg/convert2rhel/convert2rhel-secrets/-/raw/main/.env_sat_reg2
```

Comment:
This we will change to `.env_sat_reg` before merging so we do not break the testing.